### DISCUSSION TITLE
Is Lyra production ready?

### DISCUSSION THREAD

**Discussion by xyntechx** [oramasearch/orama#293] [Q&A] [answered]
Hi everyone!

I've been going through Lyra's docs and it's pretty impressive! Because of that, I'm hoping to use Lyra for my startup, [NexLiber](https://nexliber.com).

May I know whether Lyra is production-ready? Will there be breaking changes in the future we need to prepare for?

Thanks and happy coding!

**Comment by micheleriva** [comment, marked as answer]
Hi @xyntechx!
Yes, Lyra is production-ready 🙂

We're doing some research to understand how to evolve it better, would you be open for a quick call where we discuss how you would like to implement it, and what are your needs?

**Reply by xyntechx** [reply]
Hi @micheleriva! Thanks for your reply! I'll be implementing Lyra soon then :))

On the topic of improving Lyra, I was wondering whether you can integrate it with existing database solutions like Supabase (which I am currently using). Right now, I figured a "hacky" way to use Lyra together with Supabase, which is to pull all the searchable data from Supabase and batch-insert them into Lyra (I also wonder whether it's performant/efficient to do this hahaha). However, perhaps building plugins for existing databases can help Lyra stand out from the other search tools and gain more users.

Please let me know if there's any confusion on my part! Also, yes I'm open for a call, but perhaps we should do it tomorrow night my time (UTC+8) since I'm busy during the day!

Cheers :))

**Reply by zanzlender** [reply]
I'm trying this out as well... So is this way efficient or should I do this another way? xD

**Reply by micheleriva** [reply]
>  I was wondering whether you can integrate it with existing database solutions like Supabase

We are working on this kind of solution :)

**Reply by zanzlender** [reply]
@micheleriva Since it can run on the edge I suppose Lyra is made to be performant in reading data from another source and fastly creating a database. I'm working on something and want to know if this is a good idea...

I want to use this in a React project with NodeJS as backend, so is it a good idea to have this kind of flow:

1. User can update the data to be searched, so It will be stored in a local JSON file
2. Lyra will be in a API endpoint for searching, when someone requests it, Lyra will create a database from the JSON file and return the search results

(for the moment I want the file to be local, later it will probably be somewhere in the cloud)

My main concerns are: Is this efficient or is there a better way? Is this generally a good use for Lyra?
